Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR_test.py

```diff
@@ -222,6 +264,23 @@ def test_get_incident_extra_data(requests_mock):
 
 class TestFetchStarredIncident:
 
+    def test_get_starred_incident_list(self, requests_mock):
```

Comment:
Same as above, what is this test was added for?

Reply:
I didn't add this one